triphouse/package.json

```diff
@@ -9,6 +9,8 @@
     "react": "^17.0.2",
```

Comment:
Why do you have two package.json in a project?

Reply:
there was a logical explanation for that, but for me would be easier just to delete 1 package.json :))